src/themes/createLeading.ts

```diff
@@ -0,0 +1,26 @@
+import { merge } from 'lodash'
+
+interface Leading {
+  NONE: number
+  TIGHT: number
+  NORMAL: number
+  RELAXED: number
+}
+
+export type LeadingProperty = Partial<Omit<Leading, 'NONE'>>
+export type CreatedLeading = Leading
+
+export type Leadings = 'NORMAL' | 'TIGHT' | 'RELAXED' | 'NONE'
```

Comment:
[IMO]
```suggestion
export type Leadings = keyof Leading
```
でいけそうとか思ったりしましたが、これだと微妙そうでしょうか

Reply:
全く問題ないです! 使わせてもらいます!